pxc/local/test-binary-pxc57

```diff
@@ -73,6 +73,10 @@ if [[ $MTR_ARGS == *"--big-test"* ]] || [[ $MTR_ARGS == *"--only-big-test"* ]];
     TESTCASE_TIMEOUT=$((TESTCASE_TIMEOUT * 2))
 fi
 
+if [[ $WITH_ASAN == "YES" ]]; then
```

Comment:
I think we need to export WITH_ASAN in run-test57.sh as well

Reply:
Done.